.github/dependabot.yml

```diff
@@ -8,15 +8,15 @@ updates:
       interval: monthly
     commit-message:
       prefix: meta
-    open-pull-requests-limit: 10
+    open-pull-requests-limit: ${{secrets.OPEN_PR_LIMIT}}
```

Comment:
```suggestion
    open-pull-requests-limit: ${{vars.OPEN_PR_LIMIT}}
```

Reply:
That sounds good to me, but I'm not sure it exists, I have only be able to find dependabot secrets, not variables.